src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UIPrefsAccessor.java

```diff
@@ -494,6 +494,16 @@ public String getThemeErrorClass()
                     JsArrayString.createArray().<JsArrayString>cast());
    }
 
+   public PrefValue<Integer> maxCheckWords()
```

Comment:
Does this need some code in `UIPrefs.java` to update the preference after the user has changed the option? See e.g. https://github.com/rstudio/rstudio/commit/72c67f5317134cff6911391d88a1604ec13237d8#diff-525e35862e06774c892a1991ec1784b0.

Is an associated change in `TextEditingTarget.java` also warranted (to ensure that existing documents see the updated preference value)?

Reply:
Oh maybe we do, thanks! I'll toss that in.